index.js

```diff
@@ -1,82 +1,194 @@
-
 /** Array Functions **/
 /*
 0. Write a function that receives the array below as parameters and returns a new array which has all the elements added with 2
 */
-var strArr = ['13', '2', '34', '14', '5', '86', '3.46'];
+console.log("============");
+console.log("Exercise 0:");
+
+var strArr = ["13", "2", "34", "14", "5", "86", "3.46"];
+
+const typeCastAndAdd = arr => arr.map(item => (Number(item) + 2).toString());
```

Comment:
For this situation is okay to use Number(item). Don't forget that there is also the parseInt() method which can save you in some cases:
https://stackoverflow.com/questions/4564158/what-is-the-difference-between-parseintstring-and-numberstring-in-javascript#answer-4564199

Reply:
In this situation, if I use parseInt(), the last item "3.46" will be converted to an integer and will return an "5".
Maybe it would be good to use parseFloat() instead of parseInt() or Number() ??

Reply:
yes, sorry I missed the float. parseFloat() would be better ;)